package.json

```diff
@@ -1,20 +1,25 @@
 {
   "name": "media-recorder-stream",
-  "version": "2.2.0",
```

Comment:
Could you leave the version unchanged? Easier if I bump it

Reply:
Yeah sure ^^